pipeline.py

```diff
@@ -159,6 +159,11 @@
     print("Generating Analysis CSVs...")
     data = AnalysisFile.generate(user, data, csv_by_message_output_path, csv_by_individual_output_path)
 
+    print("Writing TracedData to file...")
+    IOUtils.ensure_dirs_exist_for_file(json_output_path)
+    with open(json_output_path, "w") as f:
+        TracedDataJsonIO.export_traced_data_iterable_to_json(data, f, pretty_print=True)
```

Comment:
This makes me wonder whether - until we have proper archiving in place - we should also be putting the corresponding JSON files into drive. @as2388  thoughts?

Reply:
agreed, but too big to do here. Will do this in a future PR